core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -889,12 +859,6 @@
           + " each node with sufficient space. It defaults to"
           + " ${java.io.tmpdir}/accumulo-vfs-cache-${user.name}"),
 
-  @Interpolated
-  @Experimental
-  GENERAL_MAVEN_PROJECT_BASEDIR(AccumuloClassLoader.MAVEN_PROJECT_BASEDIR_PROPERTY_NAME,
-      AccumuloClassLoader.DEFAULT_MAVEN_PROJECT_BASEDIR_VALUE, PropertyType.ABSOLUTEPATH,
```

Comment:
If this is deleted, should probably also remove the corresponding properties from `AccumuloClassLoader.java`